static/views/TaskDetails.jsx

```diff
@@ -26,15 +26,47 @@ Progress.propTypes = {
 // local api client for now.
 const api = new Client();
 
+const LogLineItem = React.memo(({index, item, highlighted, setHighlightedLine}) => {
```

Comment:
Put this into a memoized component for a bit of extra optimization, so that the Ansi component doesn't try and re-render the line.